### PR TITLE
[CELEBORN-362][WEBSITE] Add 0.2.0 release to website.

### PR DESCRIPTION
**Pull request by FMX**
Release 0.2.0 is ready.

### CODE REVIEW

**Review by pan3793 — APPROVED** [review]

File: docs/download.md [hunk 1]
```diff
@@ -21,14 +21,12 @@ license: |
 
 No version avaiable yet.
 
-<!--
 ### 0.2.0-incubating (2022-12-15)
```
Comment (kaijchen) [inline docs/download.md:24]: Need update the date?

File: docs/download.md [hunk 2]
```diff
@@ -21,14 +21,12 @@ license: |
...
+|:-----------:|:-------------------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------:|:---------:|
+| Source Code | [src](https://www.apache.org/dyn/closer.lua/incubator/celeborn/celeborn-0.2.0-incubating/apache-celeborn-0.2.0-incubating-source.tgz) | [sha512](https://www.apache.org/dyn/closer.lua/incubator/celeborn/celeborn-0.2.0-incubating/apache-celeborn-0.2.0-incubating-src.tgz.sha512)  | [asc](https://www.apache.org/dyn/closer.lua/incubator/celeborn/celeborn-0.2.0-incubating/apache-apache-celeborn-0.2.0-incubating-src.tgz.asc) |
+| Binary      | [bin](https://www.apache.org/dyn/closer.lua/incubator/celeborn/celeborn-0.2.0-incubating/apache-celeborn-0.2.0-incubating-bin.tgz)    | [sha512](https://www.apache.org/dyn/closer.lua/incubator/celeborn/celeborn-0.2.0-incubating/apache-celeborn-0.2.0-incubating-bin.tgz.sha512)  | [asc](https://www.apache.org/dyn/closer.lua/incubator/celeborn/celeborn-0.2.0-incubating/apache-apache-celeborn-0.2.0-incubating-bin.tgz.asc) |
```
Comment (kaijchen) [inline docs/download.md:29]: Is the link correct?

```
https://dist.apache.org/repos/dist/release/incubator/celeborn/celeborn-0.2.0-incubating/
```

**Review by kaijchen — COMMENTED** [review]
Hi @FMX and @pan3793, please check these comments.

File: docs/download.md [hunk 3]
```diff
@@ -21,14 +21,12 @@ license: |
 
 No version avaiable yet.
```
Comment (kaijchen) [inline docs/download.md:22]: This line should be removed also.

File: docs/download.md (hunk 1 above)
Comment (pan3793) [reply docs/download.md:24]: This should be the ANNOUNCE date, thanks for catching

**Pull request merged**